### ISSUE TITLE
Add platform specific paths for .sound objects

### ISSUE THREAD

**Issue by Qwarr** [defold/defold#7678]
**Is your feature request related to a problem? Please describe (REQUIRED):**
When building for another platform such as android it would be nice to be able to specify a path for the `file.sound` could use instead of the default. This would be useful to quickly swap between high quality audio and compressed audio.

**Describe the solution you'd like (REQUIRED):**
Within the sound component add an option to add platform specific file paths that would automatically be set when building for different platform targets. 

**Describe alternatives you've considered (REQUIRED):**
Right now I'm copy/pasting the different files to override the currently set as the main path. Alternatively there could be a build script that swaps those files to manually set paths.

**Additional context (OPTIONAL):**
Current: 
![Defold 1 4 6](https://github.com/defold/defold/assets/2577547/58966887-9922-413e-9b93-aa61be040f3c)

Suggestion:
![Defold 1 4 6_2](https://github.com/defold/defold/assets/2577547/ad745bd0-4043-41f8-8241-dd8fb424a00b)

**Comment by JCash** [comment]
I don't think it should be per-platform, as it doesn't take into account the variances in that platform space (e.g. high/low end Android device).

And also, we should consider the fact that both of these sets of audio (images etc), might be bundled in the same bundle.

**Comment by subsoap** [comment]
Sound files need a similar system as images have for per-platform compression.
